### PR TITLE
fix(session-signup): fixed mentors showing when wrong skill is selected

### LINKED ISSUE #286: Mentor showing available with the wrong skill
Currently if a mentee chooses a skill it will show all available mentors and not just the mentors that know that skill.

The desired outcome is the mentee would only see available mentors that say they know that skill.

### PR DESCRIPTION
**Pull request by GitLukeW**
### **1. Targeted Issue** ###

Ticket # 111

### **2. Overview of Solution** ###

Changed to logic to be a filter on if the mentor includes that skill. 

### **3. Tools Used** ###

N/A

### **4. Testing Strategy** ###

Tested the app to ensure only mentors with the chosen skill are showing. 

### **5. Future Implications** ###

The code will need to be moved to the backend. 

### **6. Screenshots** ###

N/A

### **7. Code Reviewers** ###

N/A

---

Before you go...

- [ ]  I have linked the relevant issue
- [ ]  I have provided an overview of my solution
- [ ]  I have shared the tools I've used
- [ ]  I have outlined my testing strategy
- [ ]  I have considered future implications of my changes
- [ ]  I have included relevant screenshots
- [ ]  I have kept my team in the loop

---

### CODE REVIEW

**Review by nzeager — APPROVED** [review]
Great update! No changes needed. This can be merged once @esparr approves it.

**Review by esparr — APPROVED** [review]
Looks good to me :shipit:

**Pull request merged**
